### PR TITLE
Update the manifest to version 0.13.3

### PR DESCRIPTION
**Pull request by houshengbo**
# Issue to be fixed

Fixes #

## Proposed Changes

*
*
*

## Release Note

```release-note
```

### CODE REVIEW

File: cmd/manager/kodata/knative-eventing/0.13.2.yaml [hunk 1]
```diff
@@ -553,13 +553,13 @@ spec:
...
         app: eventing-controller
-        eventing.knative.dev/release: "v0.13.1"
+        eventing.knative.dev/release: "v0.13.2"
     spec:
       serviceAccountName: eventing-controller
       containers:
       - name: eventing-controller
         terminationMessagePolicy: FallbackToLogsOnError
-        image: gcr.io/knative-releases/knative.dev/eventing/cmd/controller@sha256:620f1a40ddb698bd486b50696b9647a55d5724742424d7fe911339c00d0e9b8d
+        image: knative.dev/eventing/cmd/controller
```
Comment (k4leung4) [inline cmd/manager/kodata/knative-eventing/0.13.2.yaml]: the image path seems wrong

**Comment by k4leung4** [comment]
see https://github.com/knative/eventing/issues/2745

**Comment by houshengbo** [comment]
@k4leung4 yep I actually found the issue, and informed the eventing team.
Already updated to 0.13.3 manifest.

**Pull request merged**